src/styles/blocks/form-field--big.css

```diff
@@ -0,0 +1,24 @@
+.form-field__first {
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  background-image: url(/src/images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 26px center;
+  padding-left: 62px;
+  height: 69px;
+  width: 100vw;
+  padding-block: 0;
+}
+
+.form-field__first:hover {
+  box-shadow: 0 4px 4px 0 #00000040;
+}
+
+.form-field__first:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: bold;
```

Comment:
Ensure that the 'Avenir' font family includes a bold weight. If not, the `font-weight: bold` style might not render as expected. Consider specifying a font-weight that is available in your font files.